x-pack/test/functional/services/ml/single_metric_viewer.ts

```diff
@@ -55,5 +90,32 @@ export function MachineLearningSingleMetricViewerProvider({ getService }: FtrPro
         timeout: 30 * 1000,
       });
     },
+
+    async openForecastModal() {
+      await testSubjects.click(
+        'mlSingleMetricViewerSeriesControls > mlSingleMetricViewerButtonForecast'
+      );
+      await testSubjects.existOrFail('mlModalForecast');
+    },
+
+    async closeForecastModal() {
+      await testSubjects.click('mlModalForecast > mlModalForecastButtonClose');
+      await testSubjects.missingOrFail('mlModalForecast');
+    },
+
+    async assertForeCastModalRunButtonEnabled(expectedValue: boolean) {
```

Comment:
Typo! `assertForeCastModalRunButtonEnabled` should be `assertForecastModalRunButtonEnabled`

Reply:
Fixed in 9fd1368